# When the quite boot parameter is passed print warnings and errors only.
diff --git a/src/common.sh b/src/common.sh
index 609b82d..48eba59 100644
--- a/src/common.sh
+++ b/src/common.sh
@@ -22,12 +22,12 @@ fi
 LANGFILE=${LANGDIR}/$(echo $LANG).txt
 msg() {
     message=$(translate $1)
-    echo -e " ${C_GREEN}*${C_CLEAR} $message $2"
+    [ -n $quiet ] || echo -e " ${C_GREEN}*${C_CLEAR} $message $2"
 }

 inf() {
     message=$(translate $1)
-    echo -e " ${C_CYAN}*${C_CLEAR} $message $2"
+    [ -n $quiet ] || echo -e " ${C_CYAN}*${C_CLEAR} $message $2"
 }
 debug() {
     message=$(translate $1)
@@ -43,7 +43,7 @@ err() {
 }
 translate(){
     if [ ! -f ${LANGFILE} ] ; then
-        echo $*
+        echo $*
         return 0
     fi
     word=$(cat ${LANGFILE} | grep "$*::" | head -n 1 | sed "s/^.*:://g")
--
